### PR TITLE
[rm] update instructions for windows signing

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
I did sneak in another fix the document

File: dev/doc/release-process.md [hunk 1]
```diff
@@ -140,14 +140,13 @@ in time.
 - [ ] Draft a release on GitHub.
 - [ ] Sign the Windows and MacOS packages and upload them on GitHub.
   + The Windows packages must be signed by the Inria IT security service. They
-    should be sent as a link to the binary together with its SHA256 hash in a
-    signed e-mail, via our local contact (currently `@maximedenes`).
+    should be sent as a link to the binary (via [filesender](https://filesender.renater.fr) for example)
+    together with its SHA256 hash in a signed e-mail to `dsi.securite` @ `inria.fr`
+    putting `@maximedenes` in carbon copy.
   + The MacOS packages should be signed by our own certificate, by sending them
     to `@maximedenes`. A detailed step-by-step guide can be found [on the wiki](https://github.com/coq/coq/wiki/SigningReleases).
```
Comment (Zimmi48) [inline dev/doc/release-process.md:147]: What about also fixing this?

```suggestion
  + The MacOS packages should be signed with our own certificate. A detailed step-by-step guide can be found [on the wiki](https://github.com/coq/coq/wiki/SigningReleases).
```

**Review by Zimmi48 — APPROVED** [review]

**Comment by gares** [comment]
given you did not merge yet, I sneaked in another update about the discourse forum

**Review by Zimmi48 — APPROVED** [review]
:rocket: Thanks a lot for making sure this list is kept up-to-date!

**Comment by Zimmi48** [comment]
@coqbot merge now

**Pull request merged**